.github/workflows/testing.yml

```diff
@@ -4,7 +4,30 @@ on:
   workflow_call: {}
 
 jobs:
+  setup:
+    name: Setup
+    runs-on: ubuntu-latest
+    outputs:
+      output1: ${{ steps.step3.outputs.SERVERLESS_INDEX_NAME }}
+    steps:
+      - name: Checkout code
+        id: step1
+        uses: actions/checkout@v4
+
+      - name: Install dependencies
+        id: step2
+        run: npm ci
+
+      - name: Run setup script
+        id: step3
+        env:
+          PINECONE_API_KEY: ${{ secrets.PINECONE_API_KEY }}
+        run: |
+          SERVERLESS_INDEX_NAME=$(npx ts-node ./src/integration/setup.ts | grep "SERVERLESS_INDEX_NAME=" | cut -d'=' -f2)
```

Comment:
Ah I see, we don't use the `integrationRunner` here, only for local runs.

Reply:
yaya :)